app/javascript/components/visual-settings-form/visual-settings-form.schema.js

```diff
@@ -0,0 +1,145 @@
+          name: 'view.summary__mode',
+          id: 'view.summary__mode',
+          label: __('Summary Screens'),
+          initialValue: 'dashboard',
+          options: [
+            { label: __('Dashboard'), value: 'dashboard' },
+            { label: __('Textual'), value: 'textual' },
+          ],
+        },
+      ],
+    },
+    {
+      component: componentTypes.SUB_FORM,
+      name: 'perpage-subform',
+      title: __('Default Items Per Page'),
+      fields: [
+        {
+          component: componentTypes.SELECT,
+          name: 'perpage.list',
+          id: 'perpage.list',
+          label: __('List View'),
+          initialValue: 20,
+          options: [5, 10, 20, 50, 100, 200, 500, 1000].map((n) => ({ label: n, value: n })),
+        },
+        {
+          component: componentTypes.SELECT,
+          name: 'perpage.reports',
+          id: 'perpage.reports',
+          label: __('Reports'),
+          initialValue: 20,
+          options: [5, 10, 20, 50, 100, 200, 500, 1000].map((n) => ({ label: n, value: n })),
+        },
+      ],
+    },
+    {
+      component: componentTypes.SUB_FORM,
+      name: 'topology-subform',
+      title: __('Topology Default Items in View'),
+      fields: [
+        {
+          component: componentTypes.SELECT,
+          name: 'topology.containers_max_items',
+          id: 'topology.containers_max_items',
+          label: __('Containers'),
+          options: [
+            { label: __('Unlimited'), value: 0 },
+            ...[5, 10, 20, 50, 100, 200, 500, 1000].map((n) => ({ label: n, value: n })),
+          ],
+        },
+      ],
+    },
+    {
+      component: componentTypes.SUB_FORM,
+      name: 'display-subform',
+      title: __('Miscellaneous'),
+      fields: [
+        {
+          component: componentTypes.SELECT,
+          name: 'display.startpage',
+          id: 'display.startpage',
+          label: __('Start Page'),
+          isSearchable: true,
+          loadOptions: () => API.get('/api/shortcuts?expand=resources&attributes=description,url')
+            .then(({ resources }) => resources.map(({ url, description }) => ({ value: url, label: description }))),
+        },
+        {
+          component: componentTypes.SELECT,
+          name: 'display.timezone',
+          id: 'display.timezone',
+          label: __('Timezone'),
+          isSearchable: true,
+          loadOptions: () => API.get('/api').then(({ timezones }) => timezones.map(({ name, description }) => ({ label: description, value: name }))),
+        },
+        {
+          component: componentTypes.SELECT,
+          name: 'display.locale',
+          id: 'display.locale',
+          label: __('Locale'),
+          initialValue: 'default',
+          options: [
+            { label: __('Global Default'), value: 'default' },
+            ...Object.keys(window.locales).map((value) => ({ value, label: window.locales[value].locale_data.app.locale_name[0] })),
```

Comment:
@mzazrivec @himdel the `window.locales` isn't available since the Rails6 update, where can I access it?

Reply:
Ideally, this info should come from the server.

`window.locales` is an implementation detail, and will NOT contain all locales if we ever do language packs to save 11M of data.

Reply:
I don't need the whole `window.locales` just a list of available locales and it was working before your changes.

Reply:
Yes, what I'm saying is, you can't expect JS to have a list of all available locales in the future. This should be an API.

But, for now, https://github.com/ManageIQ/manageiq-ui-classic/pull/7590 + https://github.com/ManageIQ/manageiq/pull/20963 restores the previous state where `window.locales` works as before.

Reply:
https://github.com/ManageIQ/manageiq-ui-classic/pull/7590 is merged